recipes/qtkeychain/all/conanfile.py

```diff
@@ -29,20 +31,33 @@ class PackageConan(ConanFile):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def validate(self):
+        check_min_cppstd(self, 17)
+        qt_options = self.dependencies["qt"].options
+        if self.settings.os in ("Linux", "FreeBSD") and not qt_options.with_dbus:
+            raise ConanInvalidConfiguration(f'{self.ref} requires -o="qt/*:with_dbus=True" in {self.settings.os}')
+
     def requirements(self):
-        self.requires("qt/[>=6.7 <7]", options={"with_dbus": True}, transitive_headers=True, transitive_libs=True)
-        self.requires("dbus/1.15.8", transitive_headers=True, transitive_libs=True)
+        self.requires("qt/[>=6.7 <7]", transitive_headers=True, transitive_libs=True)
+        # self.requires("dbus/1.15.8")
 
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
```

Comment:
The library needs C++17 to actually build anything QT related when it includes QT 6, removing this line lets the recipe compile when asking conan to use C++17 or greater